### DISCUSSION TITLE
importer "alternative does not exist" error

### DISCUSSION THREAD

**Discussion by IngridJSJ** [Spine-project/Spine-Toolbox#2631] [Support]
Dear spinetoolbox team, 

I am trying to import some data as follows:
![image](https://github.com/spine-tools/Spine-Toolbox/assets/47012366/0d476d5d-c881-454a-972e-fdaa76e9b3cd)
I get this error
DBAPIError while inserting alternative items: ('UNIQUE constraint failed: alternative.id',)
Could not import parameter value for 'Wind Onshore', class 'Technologies', parameter 'totalPotential': alternative 'NL' does not exist.
how do I add first an alternative NL? 
thank you very much!

**Comment by IngridJSJ** [comment]
actually now I reran the workflow and I dont have that error anymore :)

**Reply by DillonJ** [reply]
Ha, I know - my experience is that 9 times out of 10, writing an issue about something mysteriosuly solves it!!!

**Comment by soininen** [comment]
Sounds like an actual bug. We may have a "timing" error when importing stuff.

**Reply by soininen** [reply]
I failed to reproduce this (`master` and `0.8-dev` branches). Maybe it needs some other trigger than just the data&mappings from the description, or an older version of Toolbox.
